.github/workflows/release.yml

```diff
@@ -0,0 +1,38 @@
+name: release
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+    branches:
+      - 'gh-actions-release*'
+jobs:
+  create-release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Create tarball
+        run: |
+          files=(scripts/ samples/ templates/ LICENSE CHANGELOG.md README.md)
+          tar -zcf xrd-tools-${{github.ref_name}}.tgz "${files[@]}"
+
+      - name: Create checksum
+        run: |
+          filename="xrd-tools-${{github.ref_name}}.tgz"
+          sha256sum "$filename" > "$filename".sha256
+
+      - name: Create release
+        uses: softprops/action-gh-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          name: Release ${{ github.ref_name }}
+          body: |
+            Changes in this release
+            - First change
+            - Second change
+          files: xrd-tools-${{github.ref_name}}.tgz*
```

Comment:
You don't need to list the `.sha256` file here too?

Reply:
Uses glob at the end, would you prefer both files be listed explicitly?

Reply:
I think both files explicitly would be clearer (and then doesn't risk catching some other files that happen to match the glob).